plugin/src/main/kotlin/de/triplet/gradle/play/internal/Tracks.kt

```diff
@@ -1,12 +1,26 @@
 package de.triplet.gradle.play.internal
 
-internal val Track.superiors get() = Track.values().takeWhile { it != this && it != Track.ROLLOUT }
+internal val TrackType.superiors get() = TrackType.values().takeWhile { it != this && it != TrackType.ROLLOUT }
 
-internal enum class Track(val publishedName: String) {
+internal enum class TrackType(val publishedName: String) {
```

Comment:
Renamed to `TrackType` to not conflict with Google `Track`?

Reply:
Yep, got tired of fully-qualified classes here and there.